backend/subletting/models.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils import timezone
+
+
+User = get_user_model()
+
+class Offer(models.Model):
```

Comment:
I think I said that having this be a "through" model would automatically make it have composite key (User, Sublet), turns out I lied. So it might be worth having unique together constraint